core/src/test/groovy/pl/allegro/tech/embeddedelasticsearch/SynchronicitySpec.groovy

```diff
@@ -0,0 +1,38 @@
+package pl.allegro.tech.embeddedelasticsearch
+
+import spock.lang.Specification
+
+import static java.util.concurrent.TimeUnit.MINUTES
+
+class SynchronicitySpec extends Specification {
+    static final ELASTIC_VERSION = "2.2.0"
+
+    def "should not throw exception on starting embedded instance more than once from different threads"() {
+        when:
+        final server = EmbeddedElastic.builder()
+                .withElasticVersion(ELASTIC_VERSION)
+                .withStartTimeout(TEST_START_TIMEOUT_IN_MINUTES, MINUTES)
```

Comment:
As I said in previous comment: you have to specify http/transport port. Without this embedded-elasticsearch might start twice without your mechanism